.github/workflows/kommander-branch.yaml

```diff
@@ -0,0 +1,64 @@
+name: "Create Kommander Branch"
+
+on:
+ pull_request:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  get-kapps-branch-name:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'open-kommander-pr') }}
+    outputs:
+      branch_name: ${{ steps.branch-name.outputs.branch_name }}
+      escaped_branch_name: ${{ steps.escaped-branch-name.outputs.escaped_branch_name }}
+      base_branch_name: ${{ steps.base-branch-name.outputs.base_branch_name }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - id: branch-name
+        run: echo "::set-output name=branch_name::${{ github.head_ref }}"
+      - id: escaped-branch-name
+        run: echo "::set-output name=escaped_branch_name::$(echo ${{ github.head_ref }} | sed -e 's/\//\\\//g')"
+      - id: base-branch-name
+        run: echo "::set-output name=base_branch_name::${{ github.base_ref }}"
+      - name: Check output branch-name
+        run: echo ${{ steps.branch-name.outputs.branch_name }}
+      - name: Check output escaped-branch-name
+        run: echo ${{ steps.escaped-branch-name.outputs.escaped_branch_name }}
+      - name: Check output base-branch-name
+        run: echo ${{ steps.base-branch-name.outputs.base_branch_name }}
+
+  create-kommander-branch:
```

Comment:
we may be able to create the branch and PR in this action using a PAT that has access to the Kommander repo, but I think this workflow should be good as well!

Reply:
> [Deploy keys](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) can be set per repository and so are arguably more secure than using a repo scoped [Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token). Allowing the action to push with a configured deploy key will trigger on: push workflows. This makes it an alternative to using a PAT to trigger checks for pull requests. Note that you cannot use deploy keys alone to [create a pull request in a remote repository](https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#creating-pull-requests-in-a-remote-repository) because then using a PAT would become a requirement. This method only makes sense if creating a pull request in the repository where the workflow is running.

ref: https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#push-using-ssh-deploy-keys

Using a PAT would work but since this is a public repo I thought it would be safer to not use a PAT here

Reply:
gotcha yeah, makes sense to me